src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -535,8 +541,57 @@ public void StartMusic()
         Jukebox.Instance.Resume();
     }
 
+    public override void _PhysicsProcess(float delta)
```

Comment:
I think that using _PhysicsProcess is the wrong place to do this.

As the input actions here ignore cases where the GUI has eaten them.

Instead put all of the input handling to UnhandledInput, add boolean flags for what input is currently down, and apply the movement in _Process based on what input flags are active.

Reply:
Instead of flags, one vector would be better, right?

Reply:
If everything works. But often everything doesn't, so you need one flag per movement key as input can be missed etc. that will make a single movement vector able to get permanently stuck.